sdv/lite/tabular.py

```diff
@@ -257,7 +257,7 @@ def load(cls, path):
                 The loaded tabular model.
         """
         with open(path, 'rb') as f:
-            model = pickle.load(f)
+            model = pickle.load(f)  # noqa: DUO103
```

Comment:
this is something we might want to think about moving forward. Perhaps there is a more secure way to save and load models